src/frontend/Komponenter/Behandling/Brev/FrittståendeBrevMedVisning.tsx

```diff
@@ -3,8 +3,24 @@ import { byggTomRessurs, Ressurs } from '../../../App/typer/ressurs';
 import PdfVisning from '../../../Felles/Pdf/PdfVisning';
 import { IPersonopplysninger } from '../../../App/typer/personopplysninger';
 import { FrittståendeSanitybrev } from './FrittståendeSanitybrev';
+import { styled } from 'styled-components';
+import { VStack } from '@navikt/ds-react';
 
-import { HøyreKolonne, StyledBrev, VenstreKolonne } from './StyledBrev';
+const Container = styled.div`
+    display: flex;
+    gap: 1rem;
+    padding: 1rem;
+    background-color: #f2f2f2;
+
+    @media (max-width: 1400px) {
+        flex-direction: column;
+        padding: 3rem;
+    }
+`;
+
+const LikDelContainer = styled.div`
```

Comment:
```suggestion
const InnerContainer = styled.div`
```
Vet ikke hva som menes med LikDelContainer 🤔

Reply:
De tar like stor del - i dette tilfellet tar hver halvparten av plassen